### PR TITLE
Work On https://esm.sh/server

### PR DESCRIPTION
**Pull request by renhiyama**

### CODE REVIEW

File: server.ts [hunk 1]
```diff
@@ -166,8 +166,8 @@ if (import.meta.main && !isDenoDeploy) {
       "font-weight:bold;color:#007bff;",
     );
     console.log(
-      "%cThis is local version of esm.sh running on Deno 🦕.",
-      "color:gray;",
+      "%cThis is local version of esm.sh running on %cDeno 🦕%c.",
+      "color:gray;","color: green", "color:gray;"
```
Comment (ije) [inline server.ts:170]: i like this

File: server.node.js [hunk 2]
```diff
@@ -0,0 +1,251 @@
+//This Code can be run with Reejs@(Node/Bun)
```
Comment (ije) [inline server.node.js:1]: to serve this file, check https://github.com/esm-dev/esm.sh/blob/e864c7e6bf737f1e2a6499448e190f812bd56bfa/server/server_handler.go#L454

Comment (renhiyama) [reply server.node.js:1]: haha already found it myself!

**Comment by renhiyama** [comment]
I will try writing tests for reejs later. Probably after some weeks :sweat_smile: .
I am looking the code atm for changing the `bun` target forward to `node` target instead of es2022

**Comment by renhiyama** [comment]
> I will try writing tests for reejs later. Probably after some weeks sweat_smile . I am looking the code atm for changing the `bun` target forward to `node` target instead of es2022

Looks like its already using Node target.
I should publish my fixes and updates to reejs to complete this pr haha

**Comment by ije** [comment]
LGTM, thanks @renhiyama this is big!

**Review by ije — APPROVED** [review]

**Pull request merged**